model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -342,6 +342,39 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
 
     # Nodes postprocessing stage in new graph
     for op in new_graph.get_op_nodes():
+        # tensor names restoring
+        debug_dict = {}
+
+        for out_port in op.ports:
+            out_tensor_names = op.ports[out_port][1]
```

Comment:
Could you explain what describes `op.ports[out_port]`? Is it a list of items? How it is filled with data?

Reply:
`op.ports` is our internal attribute, dictionary, where keys are numbers of output ports and values are tuples with shape and tensor name: `{out_port_idx_1: (out_port_idx_1_shape, out_port_idx_1_tensor_name), out_port_idx_1: (out_port_idx_1_shape, out_port_idx_1_tensor_name)}`

The same comment added to the code.